resources/css/app.css

```
@@ -14,42 +15,140 @@
+        --color-accent-content: var(--color-flexoki-purple-100);
+        --color-accent-foreground: var(--color-ink);
     }
 }
 
+/* ----------------------------------------------------------
+   Replace Tailwind's saturated default hues with Flexoki tones
+   so any incidental `text-red-500`, `bg-blue-100`, etc. picks
+   up the muted family. Flexoki has no 500/700/900 stop —
+   500 aliases 400, 700 aliases 600, 900 aliases 800.
+   ---------------------------------------------------------- */
+@theme {
```

Stylelint (17.10.0)

[error] Unexpected unknown at-rule "@theme" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)